File: examples/nodejs/cli/version_update.ts
import { post } from './sock.js'

(async () => {
  const json = await post('/internal/version/update', {
    message: {
      "binary_url": "https://example.com/nodex-agent-1.0.0.zip",
      "path": "/tmp",
    }
  })

  console.log(json)
})()

File: examples/python/v2/cli/version_update.py
from sock import post


payload = {
    "message": {
        "binary_url": "http://example.com/nodex-agent-1.0.0.zip",
        "path": "/tmp",
    }
}

json_response = post("/internal/version/update", payload)

print("The response is as follows.\n")
print(json_response)
